test/inductor/test_cpu_select_algorithm.py

```diff
@@ -1373,12 +1373,18 @@ def forward(self, x):
     @patches
     @torch.no_grad
     @dtypes(torch.bfloat16)
-    @parametrize("batch_size", (32,))
-    @parametrize("in_features", (128, 144))
-    @parametrize("out_features", (64, 65))
+    @parametrize(
+        "batch_size",
+        (
+            17,
+            32,
+        ),
+    )
+    @parametrize("in_features", (128, 144, 1024))
+    @parametrize("out_features", (64, 65, 1024))
     def test_int8_woq_mm(self, dtype, batch_size, in_features, out_features):
         # x will be reshaped from 3d to 2d
-        second_dim_size = 8
+        second_dim_size = 1
```

Comment:
unnecessary change?

Reply:
Thanks for reviewing!

I could parameterize it, so that we'd also be able to check support for small `M` - `1` as well as another value less than `16`.

Reply:
Added, thanks!